lib/creek/styles/converter.rb

```diff
@@ -45,16 +46,18 @@ def self.call(value, type, style, options = {})
         when 'b'
           value.to_i == 1
         when 'str'
-          value
+          convert_html_entities(value)
```

Comment:
thinking about it... the decoding should be done by nokogiri so why is this an issue?